scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -1027,6 +1021,75 @@ public TerminationData killJob(JobId jobId) {
         return terminateJob(jobId, JobStatus.KILLED);
     }
 
+    public TerminationData killJobs(List<JobId> jobIds) {
+        jobIds.forEach(jobId -> jlogger.info(jobId, "killing job"));
+        List<JobData> jobDatas = lockJobs(jobIds);
+        if (jobDatas == null) {
+            return null;// TODO emptyData(jobId);
```

Comment:
I think that code with todo should not be merged into master.

Reply:
done